tasks/main.yml

```diff
@@ -20,12 +14,7 @@
     - python-numexpr
     - python-tables
     - python-pip
-    - PyYAML
+    - python-yaml
```

Comment:
The real name of the RPM package is `PyYAML`, `python-yaml` is effectively an alias.